### DISCUSSION TITLE
PHP Warning:  translations_api(): An unexpected error occurred. Something may be wrong with WordPress.org or this server&#8217;s configuration

### DISCUSSION THREAD

**Discussion by msaravanan89** [docker-library/wordpress#921] [General]
Hi Team,

I am getting below error while deploying wordpress in AWS ECS with Fargate container and EFS storage. I am using **AH00163: Apache/2.4.62 (Debian) PHP/8.2.23**. How to resolve this error. Thanks in advance!!.

10.0.1.235 - - [14/Sep/2024:12:52:20 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 2636 "-" "ELB-HealthChecker/2.0"

10.0.2.126 - - [14/Sep/2024:12:52:20 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 2636 "-" "ELB-HealthChecker/2.0"

[Sat Sep 14 12:57:24.224099 2024] [php:warn] [pid 18:tid 18] [client 10.0.1.235:52766] PHP Warning:  translations_api(): An unexpected error occurred. Something may be wrong with WordPress.org or this server&#8217;s configuration. If you continue to have problems, please try the <a href="https://wordpress.org/support/forums/">support forums</a>. (WordPress could not establish a secure connection to WordPress.org. Please contact your server administrator.) in /var/www/html/wp-includes/functions.php on line 6085

[Sat Sep 14 12:57:24.224099 2024] [php:warn] [pid 19:tid 19] [client 10.0.2.126:49884] PHP Warning:  translations_api(): An unexpected error occurred. Something may be wrong with WordPress.org or this server&#8217;s configuration. If you continue to have problems, please try the <a href="https://wordpress.org/support/forums/">support forums</a>. (WordPress could not establish a secure connection to WordPress.org. Please contact your server administrator.) in /var/www/html/wp-includes/functions.php on line 6085

**Comment by tianon** [comment]
These are just warnings - are they causing any actual issues with your WordPress deployment?  I'm not familiar with the functionality, but the warning text and a quick Google led me to https://developer.wordpress.org/reference/functions/translations_api/, and it sounds like this is functionality that uses a WordPress.org-hosted-service to perform translations and your deployment can't access it for some reason (perhaps intentionally blocking outbound internet access?)

**Reply by msaravanan89** [reply]
Thanks for the response. I will check my outbound internet access. thanks.
